### DISCUSSION TITLE
Ошибка

### DISCUSSION THREAD

**Discussion by CJIuBka** [MrZillaGold/VK2Discord#384] [Q&A] [answered]
Не может считать параметр MinLength, стоит 1
![gfgfgfg](https://user-images.githubusercontent.com/96467798/146895380-cafbd204-db54-464b-83d9-7dcc6ca3ec4f.PNG)
.

**Comment by egorprnn** [comment, marked as answer]
Отслеживается в https://github.com/MrZillaGold/VK2Discord/issues/386
